Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: ifs docker clean
+
+ifs: *.go
+	go build -o docker ./...
+
+docker:
+	docker build -t istio/ifs docker
+
+clean:
+	rm docker/istio-federation-server
```

If we take the suggestion to move dockerfile to toplevel

```suggestion
	rm istio-federation-server
```